2.4.c

```diff
@@ -0,0 +1,61 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+
+#define arraySize 10
+
+
+//�������� ���������, ������� ��������� ������ ���������� ���������� (� �������������� ������� rand �� stdlib.h), 
+//����� ����������� ��� ��� ������������� �������������� �������� ���, ��� � ������ ������� ����� ��������, ������� �������,
+//� � ����� � ������� ���� ������ �������. ��������� ������ �������� �� �������� �����.
+
+
+void swap(int* firstElement, int* secondElement)
+{
+    int transitElement = *firstElement;
+    *firstElement = *secondElement;
+    *secondElement = transitElement;
+}
+
+
+void sort(int* array)

```

Comment:
```suggestion
void partition(int* array, int size)
```
Это не сортировка, это разбиение массива
Размер массива нужно передавать в саму функцию. Тогда функция станет применима не только к массивам размера 10.